client/state/jetpack/settings/test/utils.js

```diff
@@ -163,6 +163,18 @@ describe( 'utils', () => {
 			} );
 		} );
 
+		test( 'should include post_by_email_address in sanitized settings if it equals "regenerate"', () => {
+			const settings = {
+				some_other_setting: 123,
+				post_by_email_address: 'regenerate',
+			};
+
+			expect( sanitizeSettings( settings ) ).to.eql( {
+				some_other_setting: 123,
+				post_by_email_address: 'regenerate',
+			} );
+		} );
+
```

Comment:
Maybe add a test that to verify that in the rest of the cases it will not be included.

Reply:
https://github.com/Automattic/wp-calypso/pull/23796/files#diff-d7ab1ac524d97af2351300527b5e6679R155 😬

Reply:
🤦‍♂️  as you noticed, there is already an existing test for that, so just 🚢